test/Main.hs

```diff
@@ -1,10 +1,23 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
Do we still need this?

Reply:
No, I forgot to remove it after removing `escapeLF`.

Deleted.